app/controllers/auth/TimeoutController.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.auth
+
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
+import views.html.auth.TimeoutView
+
+import javax.inject.Inject
+
+class TimeoutController @Inject()(mcc: MessagesControllerComponents, view: TimeoutView) extends FrontendController(mcc) {
+
+  def onPageLoad(): Action[AnyContent] = Action { implicit request =>
+    Ok(view())
```

Comment:
```suggestion
    Ok(view()).withNewSession
```

Reply:
That's why it wasn't logging out! Good spot!